src/PageLayout/PageLayout.jsx

```diff
@@ -12,4 +13,8 @@ function PageLayout({ children }) {
   );
 }
 
+PageLayout.propTypes = {
+  children: PropTypes.element
```

Comment:
Better go with PropTypes.node - it's gonna work for children of any kind... strings, numbers etc... anything that could be rendered